src/main/java/com/reshiftsecurity/plugins/intellij/gui/tree/view/TreeNodeCellRenderer.java

```diff
@@ -237,7 +237,9 @@ public Component getTreeCellRendererComponent(final JTree tree, final Object val
 				}
 
 				setToolTipText(bugInstanceNode.getTooltip());
-				setTitle(bugInstanceNode.getSimpleName());
+				setTitle(
+						String.format("%s:%s",
+								bugInstanceNode.getSourceFilename(), bugInstanceNode.getSourceLines()[0]));
```

Comment:
where does setTitle sink into? does it get rendered into HTML? could there be any way this is a vulnerability?